### ISSUE TITLE
Misleading Documentation

### ISSUE THREAD

**Issue by TannerJuby1** [prismicio/csharp-kit#44]
I'm trying to use this kit but there is ZERO documentation on even the basics. The README suggests that there is documentation, but it just leads you through links that takes you to the old repo. Does there exist any documentation on how to use this product?

**Comment by benembery** [comment]
Hey @TannerJuby1, are you using .NET core?

**Comment by TannerJuby1** [comment]
Yes, I believe so. It's pretty complicated, but I'm trying to write a script for Creatio which uses .NET, unfortunately I don't have access to the Creatio instance, I've just been tasked with updating this little portion in the script, so I don't have all the details of the environment. Instead, I've spun up a .NET 7 console project to write this script. 

I'm trying to use this package to retrieve a PDF from a Prismic document's link field. However, I cannot find any documentation on how to use this package or anything .NET related on Prismic's website, anything I do find is incredibly outdated referencing the old .net package.

I have installed this package (v 1.5.0) but cannot figure out how to instantiate a client and make a request to retrieve a PDF via a Document's Link field.

**Comment by benembery** [comment]
A quick heads up I've not done much in .NET for quite a while. However, I know that this library will not work with it. 

You might want to try [my port](https://github.com/adaptivewebworks/prismicio-netstandard-kit), instead it's on [NuGet](https://www.nuget.org/packages/prismicio.netstandard). The samples folder in the repo should help you get connected to your repo.

```
var api = await _prismic.GetApi();
var docType = "test_document";
var document = await api.GetByUID(docType, "document-uid");
```
 
 I hope this helps.
